timer-selector.js

```diff
@@ -479,74 +530,125 @@ function buildTimer(language) {
+  setMessage(language, 'start-later-label', 'start-time-later');
+
+  setMessage(language, 'duration-label', 'duration');
+  setMessage(language, 'duration-1d-label', 'duration-1d');
+  setMessage(language, 'duration-1w-label', 'duration-1w');
+  setMessage(language, 'duration-2w-label', 'duration-2w');
+  setMessage(language, 'duration-1y-label', 'duration-1y');
+  setMessage(language, 'duration-custom-label', 'duration-custom');
+
+  setMessage(language, 'unit-minute');
+  setMessage(language, 'unit-hour');
+  setMessage(language, 'unit-day');
+  setMessage(language, 'unit-week');
+  setMessage(language, 'unit-month');
+  setMessage(language, 'unit-year');
+
+  setMessage(language, 'messages-label', 'messages');
   document.getElementById('message-progress').placeholder = getMessage(language, 'timer-progress');
   document.getElementById('message-finished').placeholder = getMessage(language, 'timer-finished');
-  setMessage('message-progress-label', 'message-progress');
-  setMessage('message-finished-label', 'message-finished');
-
-  setMessage('advanced-label', 'advanced-section');
-  setMessage('height-label', 'height');
-  setMessage('width-label', 'width');
-  setMessage('custom-css-label', 'css-extra');
-  setMessage('template-label', 'template');
+  setMessage(language, 'message-progress-label', 'message-progress');
+  setMessage(language, 'message-finished-label', 'message-finished');
+
+  setMessage(language, 'advanced-label', 'advanced-section');
+  setMessage(language, 'height-label', 'height');
+  setMessage(language, 'width-label', 'width');
+  setMessage(language, 'custom-css-label', 'css-extra');
+  setMessage(language, 'template-label', 'template');
   document.getElementById('custom-css').placeholder = '#title {\n  color: #008080;\n}';
 
-  setMessage('build', 'build-timer');
-  setMessage('info-help');
-  setMessage('info-source');
+  setMessage(language, 'build', 'build-timer');
+  setMessage(language, 'copy', 'build-and-copy-timer');
+  setMessage(language, 'info-help');
+  setMessage(language, 'info-source');
+
+  initializeSummaryDeletionMessages(language);
+}
+
+function initializeDeletionScore(language, deletionScore) {
+  // language currently unused
+  var scoreBox = document.getElementById('deletion-score-value');
+  scoreBox.value = deletionScore;
+  scoreBox.onclick = scoreBox.onblur = function() {
+    if (Number(scoreBox.value) > deletionScore) {
+      scoreBox.style.backgroundColor = "yellow";
+    } else {
+      scoreBox.style.backgroundColor = "white";
+    }
+  }
 }
 
 function initializeHooks(language) {
   document.getElementById('timer-type-generic').onclick = function () {
     document.getElementById('message-progress').value = '';
     document.getElementById('message-finished').value = '';
     document.getElementById('template').value = '%%iframe%%';
+    var deletionOptElement = document.getElementById('deletion-options');
+    deletionOptElement.classList.add('hidden');
```

Comment:
Suggestion: make show/hide this element a function, like `showDeletionOptions(state) /* true or false */`